src/ApimEventProcessor/Helpers/AppConfig.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using Moesif.Api;
+using Moesif.Api.Exceptions;
+using System.Threading.Tasks;
+
+namespace ApimEventProcessor.Helpers
+{
+    public class AppConfig
+    {
+        public async Task<Moesif.Api.Http.Response.HttpStringResponse> getConfig(MoesifApiClient client, ILogger logger)
+        {
+            // Get Config
+            Moesif.Api.Http.Response.HttpStringResponse appConfig;
+            try
+            {
+                appConfig = await client.Api.GetAppConfigAsync();
+                logger.LogDebug("Application Config fetched Successfully");
+            }
+            catch (APIException inst)
+            {
+                appConfig = null;
+                if (401 <= inst.ResponseCode && inst.ResponseCode <= 403)
```

Comment:
No chance that API exception might occur while status code is less than 400? Sorry havent looked at core library. 
Also, do we want to logdebug in other cases?

Reply:
It's greater than equal to 401, there would be an exception (Unauthorized) in case when incorrect applicationId is used. Have changed it to log as error message in case of exception.